barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.hpp

```diff
@@ -193,6 +193,27 @@ struct AcirProgramStack {
     void pop_back() { witness_stack.pop_back(); }
 };
 
+struct ProgramMetadata {
+
+    // An IVC instance; needed to construct a circuit from IVC recursion constraints
+    std::shared_ptr<ClientIVC> ivc = nullptr;
+
+    bool recursive = false; // Specifies whether a prover that produces SNARK recursion friendly proofs should be used.
```

Comment:
this comment is just copied over from whoever wrote it (Akosh?) - not entirely sure its correct / makes sense